script.js

```diff
@@ -78,7 +79,7 @@ const optionSections = {
     "movement": ["translationhori", "translationverti"],
     "waviness": ["jitter", "wavinessphori", "wavinessahori", "wavinesspverti", "wavinessaverti"],
     "fade": ["revealspeed", "fadeinspeed", "fadeoutspeed", "fadeoutstart", "sawtoothfadeoutsize", "sawtoothfadeoutstart"],
-    "line": ["segments", "skipchance", "thickness", "linecap", "linered", "linegreen", "lineblue", "lineopacity", "blendmode", "shadowblur"],
+    "line": ["segments", "skipchance", "thickness", "linecap", "linered", "linegreen", "lineblue", "hueshiftspeed", "lineopacity", "blendmode", "shadowblur"],
```

Comment:
Perhaps place `hueshiftspeed` behind `lineopacity` – to me, RGB and opacity/alpha should be together.

Reply:
It make sense, Changed!